frontend/components/bridge/utils.ts

```diff
@@ -41,19 +47,86 @@ const getFromToken = (
   return fromChainConfig[tokenSymbol].address as Address;
 };
 
+/**
+ *
+ * @warning A HOOK THAT SHOULD NEVER EXIST.
+ * @deprecated TODO: This hook is planned for removal in future version
+ *
```

Comment:
I think the message is too harsh and irritated 😄 can we do this:

```suggestion
/**
 *
 * @deprecated TODO: This hook is used because BE doesn't support monthly_gas_estimate in the refill requirements yet. Remove the hook once it's supported
 *
```